doc/api/esm.md

```diff
@@ -448,7 +418,49 @@ const cjs = require('./cjs.cjs');
 cjs === 'cjs'; // true
 ```
 
-## Experimental JSON modules
+### Differences between ES modules and CommonJS
+
+#### No `NODE_PATH`
+
+`NODE_PATH` is not part of resolving `import` specifiers. Please use symlinks
+if this behavior is desired.
+
+#### No `require`, `exports`, `module.exports`, `__filename`, `__dirname`
+
+These CommonJS variables are not available in ES modules.
+
+`require` can be imported into an ES module using [`module.createRequire()`][].
+
+Equivalents of `__filename` and `__dirname` can be created inside of each file
+via [`import.meta.url`][].
+
+```js
+import { fileURLToPath } from 'url';
+import { dirname } from 'path';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = dirname(__filename);
+```
+
+#### No `require.resolve`
+
+Former use cases relying on `require.resolve` to determine the resolved path
+of a module can be supported via [`import.meta.resolve`][].
+
+#### No `require.extensions`
+
+`require.extensions` is not used by `import`. The expectation is that loader
+hooks can provide this workflow in the future.
+
+#### No `require.cache`
+
+`require.cache` is not used by `import`. It has a separate cache.
+
+<i id="#esm_experimental_json_modules"></i>
+
```

Comment:
```suggestion
<i id="esm_experimental_json_modules"></i>
```

Reply:
Also if you can fix the other anchors:

https://github.com/nodejs/node/blob/adae822035b640531044b33892c32a3a3011cf5b/doc/api/esm.md#L73-L76

Reply:
Thanks, I've fixed up all 6 cases of this (including the previously existing ones).